mkt/site/utils.py

```diff
@@ -265,11 +264,11 @@ def __iter__(self):
             return iter([])
 
         chunk = 4096
-        fp = storage.open(self.path, 'rb')
+        fp = private_storage.open(self.path, 'rb')
         if 'wsgi.file_wrapper' in self.request.META:
             return self.request.META['wsgi.file_wrapper'](fp, chunk)
         else:
-            self['Content-Length'] = storage.size(self.path)
+            self['Content-Length'] = private_storage.size(self.path)
```

Comment:
I wonder if this should be `local_storage` here after my patch. `HttpResponseSendFile` should never be called if we are using S3.